AutomatedNotebooks-Manager.ipynb

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    incident_file = open("incident_archive", "w")
</pre><p>Maybe enclose the cell code in in a with open.... context manager.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>